lib/Steps/ChooseIntegration.ts

```diff
@@ -1,30 +1,49 @@
 import type { Answers } from 'inquirer';
 import { prompt } from 'inquirer';
 
+import { readFileSync } from 'node:fs';
+import { hasPackageInstalled } from '../../src/utils/package-json';
 import {
   Args,
   DEFAULT_URL,
   getIntegrationChoices,
   Integration,
 } from '../Constants';
+import { dim, red } from '../Helper/Logging';
 import { BaseStep } from './BaseStep';
 import { Cordova } from './Integrations/Cordova';
 import { Electron } from './Integrations/Electron';
-import { hasPackageInstalled } from '../../src/utils/package-json';
-import { dim } from '../Helper/Logging';
-import { readFileSync } from 'node:fs';
 
 let projectPackage: Record<string, unknown> = {};
 
-try {
+const projectPackagePathCandidates = [
   // If we run directly in setup-wizard
-  projectPackage = JSON.parse(
-    readFileSync('../../package.json', 'utf-8'),
-  ) as Record<string, unknown>;
-} catch {
-  projectPackage = JSON.parse(
-    readFileSync(`${process.cwd()}/package.json`, 'utf-8'),
-  ) as Record<string, unknown>;
+  '../../package.json',
+
+  // If we run from the CLI
+  `${process.cwd()}/package.json`,
+];
+
+for (const pathCandidate of projectPackagePathCandidates) {
+  let data: string;
+  try {
+    data = readFileSync(pathCandidate, 'utf-8');
+  } catch (error) {
+    // If the file does not exist, continue to the next candidate
```

Comment:
Nit: file not existing is `ENOENT`. On rare occasions you may actually have the file but cannot read it for some reason (`EACCESS` for instance).

I think this is fine as it is but we may wanna add a note or try to distinguish `ENOENT` from others.